chart/templates/iq-server-deployment.yaml

```diff
@@ -218,7 +218,14 @@ spec:
       initContainers:
         - name: {{ .Release.Name }}-set-iq-persistence-ownership
           image: busybox:1.28
-          command: ["sh", "-c", "chown -R 1000:1000 {{ .Values.iq_server.config.clusterDirectory }}"]
+          command:
+            - "sh"
+            - "-c"
+            - {{- if .Values.iq_server.pvOwnershipOverride }}
+              {{ .Values.iq_server.pvOwnershipOverride }}
+              {{- else }}
+              chown -R 1000:1000 {{ .Values.iq_server.config.clusterDirectory }}
```

Comment:
this doesn't need to be wrapped in quotation marks like it was before?

Reply:
using the common sintaxis, it is not necessary.